test/integration/MCR/updateMCR.js

```diff
@@ -41,6 +41,7 @@ async function updateMCRSetup() {
 
   const operator = await tk.operator();
   await setEtherBalance(operator, parseEther('10000000'));
+  await setEtherBalance(member1.address, parseEther('1000000'));
```

Comment:
not sure why this integration is suddenly failing with "not enough funds" when ran along with the whole test suite.

```
  1) updateMCR
       increases desiredMCR:
     InvalidInputError: sender doesn't have enough funds to send tx. The max upfront cost is: 100000029026712406373968 and the sender's account only has: 9999999323294995263065

```

the test it does not seem to fail when ran on its own.

All the changes in this PR are confined within unit tests / viewer contracts so not sure how its affecting that particular integration test. but adding more funds to the address seems to fix it.

https://github.com/NexusMutual/smart-contracts/actions/runs/9870557531/job/27257740964